user_manual/pim/sync_thunderbird.rst

```diff
@@ -53,11 +53,11 @@ The old method: Manually subscribing to calendars
 This method is only needed if you don't want to install TBSync.
 
 1. Go to your Nextcloud Calendar and click on the 3 dotted menu for the calendar that you want to synchronize which will display and URL that looks something like this:
-https://cloud.nextcloud.com/remote.php/dav/calendars/daniel/personal/
+https://cloud.nextcloud.com/remote.php/dav/calendars/daniel/personal/ .
```

Comment:
why would you add a point to the URL?

Reply:
This is not an added period in the address, but outside of it. It is a complete sentence and should be ended with a full stop or punctuation mark. A sentence without a period looks incomplete.

Reply:
I wanted to point out that it might be mistaken for a part of the URL....

Reply:
> I wanted to point out that it might be mistaken for a part of the URL....

Yes, I know it, but that's how the sentence ends. See other user manuals.

I checked this link again and it doesn't really refer to another website. I changed that as an example without a link and it should be fine now.